module/platform/Webots/src/Webots.hpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2021 NUbots <nubots@nubots.net>
+ */
+
+#ifndef MODULE_PLATFORM_WEBOTS_HPP
+#define MODULE_PLATFORM_WEBOTS_HPP
+
+#include <nuclear>
+
+#include "message/platform/webots/messages.hpp"
+#include "message/platform/webots/ConnectRequest.hpp"
+
+namespace module::platform {
+
+class Webots : public NUClear::Reactor {
+private:
+    /// @brief Send the inital message that tells webots who we are. We should do this when we first connect or when we
+    /// want to reconnect.
+	/// @param The socket we want to send our details to.
```

Comment:
You're meant to put the parameter name when doing this javadoc style `@param` stuff. [It looks like you're also "meant" to have an `@param` for every single parameter of every method, (see "required tags" on this link)](https://www.oracle.com/au/technical-resources/articles/java/javadoc-tool.html)
```suggestion
    /// @param fd The socket we want to send our details to.
```